physics/eclipse_test.cpp

```diff
@@ -38,16 +39,14 @@ class EclipseTest : public testing::Test {
             "initial_state_jd_2433282_500000000.proto.txt");
   }
 
-  void CheckLunarEclipse(Instant const& current_time,
-                         Sign const moon_offset_sign) {
+  void CheckLunarUmbralEclipse(Instant const& current_time,
+                               Sign const moon_offset_sign) {
     auto ephemeris = solar_system_1950_.MakeEphemeris(
         McLachlanAtela1992Order5Optimal<Position<ICRFJ2000Equator>>(),
         45 * Minute, 5 * Milli(Metre));
 
-    ephemeris->Prolong(
-        JulianDate(2433374.5));  // Prolong just past date of eclipse
-                                 // (Eclipse was 1950-04-02 but JD
-                                 // is 1950-04-03 00:00:00).
+    ephemeris->Prolong(current_time +
+                       86400 * Second);  // Prolong 1 day past date of eclipse.
```

Comment:
Please use `1 * Day` (`Day` is defined in `si.hpp`).  You won't believe how easy it is to type `84600` instead of the correct value, and that's impossible to detect at code review time.